libs/libc/signal/sig_addset.c

```diff
@@ -24,6 +24,7 @@
 
 #include <signal.h>
```

Comment:
Minor: I think we can remove `#include <signal.h>` as `#include <nuttx/signal.h>` should be enough

Reply:
Done

Reply:
Yes, but fully violated in most of the kernel code. Please check `sched/semaphore`. Most of the `sem_xxx.c` files in that folder includes only `semaphore/semaphore.h` and not `nuttx/semaphore.h` or `semaphore.h`.

Reply:
Let's do things right here